games-board/xboard/xboard-4.9.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils fdo-mime gnome2-utils
```

Comment:
I don't see eutils being used in here, is it? Also fdo-mime is deprecated, switch to xdg or xdg-utils.